xml/CRSChannelCode.xml

```diff
@@ -129,6 +137,10 @@
                 <narrative>Agency for International Trade Information and Co-operation</narrative>
                 <narrative xml:lang="fr">Agence de coopération et d'information pour le commerce international</narrative>
             </name>
+            <acronym>
+                <narrative>AITIC</narrative>
+                <narrative xml:lang="fr">ACICI</narrative>
+            </acronym>
```

Comment:
Looks like they've added a new element `<acronym>` to the file which has been pulled in. This is causing our Schema validation to fail. Should I remove this?

Reply:
Thanks Nik, please can we ignore all of the new `<acronym>` elements. We don't include these within our codelist.